server/views/pages/component.njk

```diff
@@ -204,6 +204,14 @@
             {% endif %}
           </td>
         </tr>
+        <tr>
+          <th>Trivy Scan Results</th>
+          <td>
+            <a class="govuk-link--no-visited-state" href="/trivy-scans/{{component.name}}">
+              <img src="/assets/images/trivy.png" alt="Trivy Logo" style="width: 32px; height: 32px; margin-right: 5px;" />
```

Comment:
Create a class in your CSS file, definitely try not using inline CSS if you can.